tests/sandbox/create_message_batches/test_field_validation.py

```diff
@@ -450,7 +558,19 @@ def test_invalid_message_reference(nhsd_apim_proxy_url, correlation_id):
 @pytest.mark.parametrize("correlation_id", CORRELATION_IDS)
 def test_blank_value_under_messages(nhsd_apim_proxy_url, invalid_value, correlation_id):
     """
-    .. py:function:: Test invalid messages value
+    .. py:function:: Scenario: A user submitting a request with an invalid \
+        message value receives a 400 'Invalid Value' response
+
```

Comment:
List of invalid values being tested?

Reply:
Addressed in request_properties.rst, linked below given/when/then